src/components/RepoList/RepoList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './RepoList.module.scss';
+import classNames from 'classnames/bind';
+import RepoItem from '../RepoItem';
+import { Repo } from '../../modules/repo';
+
+const cx = classNames.bind(styles);
+
+interface Props {
+  repos: Repo[];
+}
+
+const RepoList: React.FC<Props> = ({ repos }) => {
+  return (
+    <ul className={cx('repo-list')}>
+      {
+        repos.map((repo, index) => <RepoItem key={index} repo={repo} />)
```

Comment:
response에 각 레포마다 `id` 도 있던데 `key` 를 `id`를 주면 어떨까요?! ㅎㅎ

```suggestion
        repos.map(({id, ...repo }, index) => <RepoItem key={id} repo={repo} />)
```

https://ko.reactjs.org/docs/lists-and-keys.html#keys

Reply:
오 희라님 의견 동의합니다.
추가로 index params 또한 생략 가능하겠네요! 

```suggestion
        repos.map((repo) => <RepoItem key={repo.id} repo={repo} />)
```

Reply:
아 그렇네요. 지금 확인해보니 github api의 리턴값으로 id속성이 있네요. 감사합니다!